example/lib/screens/component/multi_component/multi_component_advanced_screen.dart

```diff
@@ -212,36 +212,6 @@ class MultiComponentAdvancedScreen extends StatelessWidget {
           type: ApplePaySummaryItemType.definite,
         ),
       ],
-      requiredShippingContactFields: [
-        ApplePayContactField.name,
-        ApplePayContactField.postalAddress,
-      ],
-      shippingContact: ApplePayContact(
-        givenName: "John",
-        familyName: "Doe",
-        addressLines: ["Simon Carmiggeltstraat 6"],
-        postalCode: "1011 DJ",
-        city: "Amsterdam",
-        country: "Netherlands",
-      ),
-      applePayShippingType: ApplePayShippingType.shipping,
-      allowShippingContactEditing: true,
-      shippingMethods: [
-        ApplePayShippingMethod(
-          label: "Standard shipping",
-          detail: "DHL",
-          amount: Amount(value: 1000, currency: "EUR"),
-          identifier: "identifier 1",
-          startDate: DateTime.now().add(const Duration(days: 2)),
-          endDate: DateTime.now().add(const Duration(days: 5)),
-        ),
-        ApplePayShippingMethod(
-          label: "Store pick up",
-          detail: "Weekdays, from 9:00 am to 6:00 pm",
-          amount: Amount(value: 0, currency: "EUR"),
-          identifier: "identifier 2",
-        ),
-      ],
```

Comment:
Why remove this?

Reply:
Because I wanted to have an example without extra and decided to use the multi component screen for this use-case. The example screen for the advanced flow still contains the extra fields.